src/index.html

```diff
@@ -152,18 +176,67 @@ <h2 class="modal__title">New post</h2>
         </form>
       </div>
     </div>
+
     <div class="modal" id="preview-modal">
       <div class="modal__content modal__content_type_preview">
         <button
           aria-label="Close button"
           type="button"
           class="modal__close modal__close_type_preview"
         ></button>
+
         <img class="modal__image" src="" alt="" />
         <p class="modal__caption"></p>
       </div>
     </div>
-    <script defer src="./scripts/validation.js"></script>
-    <script defer src="./scripts/index.js"></script>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__content">
```

Comment:
* Please, make the `avatar` modal match the `mobile` design https://snipboard.io/E1HAjD.jpg 


Please, add `modal__content_type_avatar` to change the styles of the container